estimatey-api/Estimatey.Application/DeveloperTime/Queries/ListLoggedTimeByDeveloper/ListLoggedTimeByDeveloperQuery.cs

```diff
@@ -58,11 +62,29 @@ public class ListLoggedTimeByDeveloperQueryHandler : IRequestHandler<ListLoggedT
 
     private async Task<List<LoggedTimeDto>> GetLoggedDeveloperTime(ProjectEntity project)
     {
-        var allLoggedTime = await _floatClient.GetLoggedTime(project.FloatId);
-
-        return allLoggedTime
+        var historicLoggedTime = await _dbContext.LoggedTime
+            .Where(_ => !_excludedFloatPersonIds.Contains(_.FloatPerson.FloatId))
+            .Select(_ => new LoggedTimeDto()
+            {
+                Id = _.FloatId,
+                PersonId = _.FloatPerson.FloatId,
+                Date = _.Date.ToString("yyyy-MM-dd"),
+                Hours = _.Hours,
+                Locked = _.Locked,
+                LockedDate = _.LockedDate == null ? null :_.LockedDate.Value.ToString("yyyy-MM-dd"),
+            })
+            .ToListAsync();
+
+        var recentLoggedTime = await _floatClient.GetLoggedTime(
+            project.FloatId,
+            project.LoggedTimeHasBeenSyncedUntil?.AddDays(1),
+            DateOnly.FromDateTime(_dateTimeService.Now));
```

Comment:
Minor but wonder if it makes sense to have a `_dateTimeService.NowDateOnly` property or something?

Reply:
Yeah was thinking this also but didn't bother, will do it!